fbw-a32nx/src/systems/fmgc/src/navigation/Navigation.ts

```diff
@@ -114,6 +117,14 @@ export class Navigation implements NavigationProvider {
     (_, i) => `L:A32NX_ADIRS_ADR_${i + 1}_STATIC_AIR_TEMPERATURE`,
   );
 
+  private static readonly irDiscreteWordVars = Array.from(
```

Comment:
A bit off-topic, but do you think it would be worth using `SimVar.GetRegisteredId` for these? I've never used it but remember you saying there's a perf benefit.

Reply:
I started off with that, but it would require adding overloads to the Arinc429Register and friends too. Could be a future improvement.